### PR TITLE
feat: enqueue db import jobs in staging & prod

### PR DESCRIPTION
**Pull request by jgadling**

### CODE REVIEW

File: ingestion_tools/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ WORKDIR /usr/src/app
 RUN mkdir ingestion_tools
 
 # Base utilities
-RUN apt update && apt install -y wget
+RUN apt update && apt install -y wget unzip jq
```
Comment (manasaV3) [inline ingestion_tools/Dockerfile:7]: Whats using `jq`?

File: ingestion_tools/infra/sync-v0.0.1.wdl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        String flags
+    }
+
+    command <<<
+        set -euxo pipefail
+        export PYTHONUNBUFFERED=1
+        python --version 1>&2
+        ls -l 1>&2
+        pwd 1>&2
+        cd /usr/src/app/ingestion_tools/scripts
```
Comment (manasaV3) [inline ingestion_tools/infra/sync-v0.0.1.wdl:20]: Are all of these still needed?

File: ingestion_tools/scripts/db_import.py [hunk 3]
```diff
@@ -23,19 +23,40 @@ def cli():
...
-    is_flag=True,
-    required=True,
-    default=False,
-    type=bool,
-    help="Use anonymous access to S3",
-)
+@click.option("--filter-dataset", type=str, default=None, multiple=True)
+@click.option("--s3-prefix", required=True, default="", type=str)
+@click.option("--endpoint-url", type=str, default=None)
 @click.option(
```
Comment (manasaV3) [inline ingestion_tools/scripts/db_import.py:60]: What are your thoughts on adding `--debug` to the db_import_options and having it default to True for any time we run db_import in staging/prod?

File: ingestion_tools/infra/sync-v0.0.1.wdl [hunk 4]
```diff
@@ -0,0 +1,54 @@
+version 1.0
+
+task cryoet_data_sync_workflow {
```
Comment (manasaV3) [inline ingestion_tools/infra/sync-v0.0.1.wdl:3]: 💜

File: ingestion_tools/infra/sync-v0.0.1.wdl (hunk 2 above)
Comment (jgadling) [reply ingestion_tools/infra/sync-v0.0.1.wdl:20]: This file isn't needed yet -- it's for a Coming Soon ™️ feature

File: ingestion_tools/scripts/db_import.py (hunk 3 above)
Comment (jgadling) [reply ingestion_tools/scripts/db_import.py:60]: good call! added that.

**Review by manasaV3 — APPROVED** [review]
🚢

**Pull request merged**